lightning/src/ln/functional_tests.rs

```diff
@@ -4147,14 +4148,14 @@ fn test_onchain_to_onchain_claim() {
 	let commitment_tx = nodes[0].node.channel_state.lock().unwrap().by_id.get_mut(&chan_1.2).unwrap().channel_monitor().get_latest_local_commitment_txn();
 	nodes[1].block_notifier.block_connected(&Block { header, txdata: vec![commitment_tx[0].clone()]}, 1);
 	let b_txn = nodes[1].tx_broadcaster.txn_broadcasted.lock().unwrap();
-	assert_eq!(b_txn.len(), 4);
-	check_spends!(b_txn[1], chan_1.3); // Local commitment tx, issued by ChannelManager
-	check_spends!(b_txn[2], b_txn[1]); // HTLC-Success tx, as a part of the local txn rebroadcast by ChannelManager in the force close
-	assert_eq!(b_txn[0], b_txn[3]); // HTLC-Success tx, issued by ChannelMonitor, * 2 due to block rescan
+	// ChannelMonitor: claim tx, ChannelManager: local commitment tx
```

Comment:
This says 2, but then we check for 3?

Reply:
Oops, check is false, comment right, deleted too much. Should be fix